db/schema.rb

```diff
@@ -10,194 +10,187 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170330225217) do
+ActiveRecord::Schema.define(version: 20170610162757) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "hstore"
 
-  create_table "activities", force: :cascade do |t|
-    t.integer  "team_id"
-    t.string   "kind",         limit: 255
-    t.string   "guid",         limit: 255
-    t.string   "author",       limit: 255
-    t.string   "title",        limit: 255
-    t.text     "content"
-    t.string   "source_url",   limit: 255
+  create_table "activities", id: :serial, force: :cascade do |t|
```

Comment:
No, should not be necessary to explicitly cite the ID type, unless you need something different like`UUID` of course. From what I remember AR just adds the information based on the underlying adapter and its version *(in my case 9.6.3)*

So for your second question: technically speaking, it is no problem to run the app on Postgres versions < 9.5 as long as the people doing it do not always commit their changed `schema.rb` if they did not add any new migrations themselves. Therefore, in order to prevent a row of commits, e.g. removing the `limit` option on every string column, etc. I'm gonna update the Readme to require a Postgres version approxing the one run on production (9.5) for development 😉 
Good point, thanx!